packages/local_auth/example/ios/RunnerTests/FLTLocalAuthPluginTests.m

```diff
@@ -56,13 +56,15 @@ - (void)testSuccessfullAuthWithBiometrics {
                                                               arguments:@{
                                                                 @"biometricOnly" : @(YES),
                                                                 @"localizedReason" : reason,
+                                                                @"localizedFallbackTitle" : @"",
```

Comment:
Shouldn't we have a test where `localizedFallbackTitle` isn't defined?  And if it isn't we'd want `OCMVerify([mockAuthContext setLocalizedFallbackTitle:@""]);` right?  That isn't how it would turn on the way the code is written.  It'd send in `nil`.

Reply:
`nil` is correct:

> Use the localizedFallbackTitle property to set a title for the fallback button. When the property is nil—as it is by default—the user sees an appropriate default title, like “Use Passcode”.

(From Apple's docs)

Reply:
But you're right about the test cases; we should be not sending anything in the normal tests, because that's what the Dart code does when there's no fallback. I missed that.